ledger/ledger-api-integration-tests/src/test/itsuite/scala/com/digitalasset/platform/tests/integration/ledger/api/TransactionServiceIT.scala

```diff
@@ -695,6 +687,45 @@ class TransactionServiceIT
           resultF.map(_.agreementText shouldBe Some(""))
       }
 
+      "do not expose the contract key in CreatedEvents for templates that do not have them" in allFixtures {
+        c =>
+          val resultF = c.submitCreate(
+            "Creating_CallablePayout_contract_for_contract_key_test",
```

Comment:
Please add the `runSuffix` to the commandId, so this test can run multiple times on the same running ledger. Example: https://github.com/digital-asset/daml/blob/33bb02ec24e7c76cbf49a747562e6628a9137a99/ledger/ledger-api-integration-tests/src/test/itsuite/scala/com/digitalasset/platform/tests/integration/ledger/api/TransactionServiceIT.scala#L328

Reply:
Addressed by https://github.com/digital-asset/daml/pull/1586/commits/f8650fb934fa36936a8a97940befe95ecd600839